.travis.yml

```diff
@@ -15,8 +22,8 @@ before_script:
 - npm run test
 - npm run example &
```

Comment:
I do not get this command working only for OGC tests...
I tried the same way than the other being different with the & is giving errors
if [[ "$TESTS" = "OGC_TESTS" ]]; then npm run example & ; fi

But The command "if [[ "$TESTS" = "OGC_TESTS" ]]; then npm run example & ; fi" failed and exited with 1

Trying to get a solution, any idea @ferjm?
Thanks!